src/JWTVerifier.php

```diff
@@ -125,7 +131,7 @@ public function verifyAndDecode($jwt)
                 throw new CoreException("We can't trust on a token issued by: `{$body->iss}`.");
             }
 
-            $secret = $this->JWKFetcher->fetchKeys($body->iss);
+            $secret = $this->JWKFetcher->fetchKeys($body->iss.$this->jwks_path);
```

Comment:
let's move the `$body->iss.$this->jwks_path` one line above to something like `jwks_url` for clarity